vm/actor/tests/set_multimap_test.rs

```diff
@@ -2,43 +2,43 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use actor::{u64_key, SetMultimap};
-use address::Address;
+use clock::ChainEpoch;
 
 #[test]
 fn put_remove() {
     let store = db::MemoryDB::default();
     let mut smm = SetMultimap::new(&store);
 
-    let addr = Address::new_id(100);
-    assert_eq!(smm.get(&addr), Ok(None));
+    let epoch: ChainEpoch = 100;
+    assert_eq!(smm.get(u64_key(epoch)), Ok(None));
 
-    smm.put(&addr, 8).unwrap();
-    smm.put(&addr, 2).unwrap();
-    smm.remove(&addr, 2).unwrap();
+    smm.put(epoch, 8).unwrap();
+    smm.put(epoch, 2).unwrap();
+    smm.remove(epoch, 2).unwrap();
 
-    let set = smm.get(&addr).unwrap().unwrap();
+    let set = smm.get(u64_key(epoch)).unwrap().unwrap();
```

Comment:
indicating this is the reason why it shouldn't (should just be able to pass through the epoch)